edb/lang/schema/schema.py

```diff
@@ -286,6 +287,10 @@ def _get(self, name, *, getter, default, module_aliases):
             if result is not None:
                 return result
 
+        # if nothing was found, check if it's the special anytype
+        if name == 'anytype':
+            return s_pseudo.Any.create()
```

Comment:
`Schema.get()` shouldn't resolve `anytype`. It's not in the schema.

Reply:
You're right, it's a remnant from an attempt to make `Expr IS anytype` work. I'll remove it for now.